### DISCUSSION TITLE
Assets.load apparently can't parse image

### DISCUSSION THREAD

**Discussion by steven-twerdochlib** [pixijs/pixijs#10751] [Q&A] [answered]
Hi I'm trying to upgrade from pixi v7 to pixi v8. I pixi v7 I had the following code:
`const texture = await PIXI.Texture.fromURL(imageUrl);`
But the migration guide says I now need to add the following line of code before that:
`await Assets.load(imageUrl);`
However now I'm getting the error: " blah blah blah... could not be loaded as we don't know how to parse it, ensure the correct parser has been added".
I'm completely lost on how to add a parser or declare one, can anyone help me?

**Comment by bigtimebuddy** [comment, marked as answer]
For images without a recognizable file extension or not a base64 encoded string, you need to set the `loadParser` like this:

```js
const texture = await Assets.load({
  src: imageUrl,
  loadParser: "loadTextures"
});
```
